templates/experiments/chat/input_bar.html

```diff
@@ -3,7 +3,7 @@
 {% else %}
   {% url 'experiments:experiment_session_message' team_slug=request.team.slug experiment_id=experiment.public_id session_id=session.external_id version_number=session.get_experiment_version_number as message_url%}
 {% endif %}
-<form class="w-full p-2 bg-gray-300"
+<form class="w-full p-2 bg-gray-300 dark:bg-gray-800"
```

Comment:
```suggestion
<form class="w-full p-2 bg-base-300"
```

Reply:
I made the suggested changes [here](https://github.com/dimagi/open-chat-studio/pull/1523/commits/f3080825b5d19b7d74acbc943c438f344e500de7)